tests/integration_tests/run_integration_tests.py

```diff
@@ -62,8 +57,7 @@ def main():
 
             if "--with-coverage" in sys.argv:
                 args = [
-                    sys.executable,
-                    coverage_path,
+                    "coverage",
```

Comment:
Everything below this and [requirements.txt](https://github.com/shotgunsoftware/tk-core/pull/678/files#diff-b3017d65db7d222a0f15767d7d0ec854) is just files being removed because I removed the `coverage` module.